crates/lib3h/src/transport/memory_mock/memory_server.rs

```diff
@@ -94,28 +97,32 @@ impl MemoryServer {
         if res.is_some() {
             return Err(TransportError::new("connectionId already used".to_string()));
         }
-        // Notify our TransportMemory to connect back
-        self.new_conn_inbox.push(requester_uri.to_string());
-        self.connection_ids.insert(requester_uri.to_string());
+        // Notify our TransportMemory (so it can connect back)
+        self.connection_inbox
+            .push((requester_uri.to_string(), true));
```

Comment:
could convert from tuple to struct for clarity on what the bool means, but not a big deal

Reply:
how about just typedefs for true and false?